src/components/e-textarea.vue

```diff
@@ -95,10 +105,10 @@
        *
        * @returns  {Object}   BEM classes
        */
-      modifiers() {
+      modifiers(): IModifiers {
         return {
           ...this.stateModifiers,
-          notification: Boolean(this.$props.notification && this.hasFocus),
+          notification: this.notification && this.focus,
```

Comment:
`notification` could be null, which would result in a non-boolean. Please wrap in `!!(...)`